themes/powerline/powerline.theme.bash

```diff
@@ -2,6 +2,8 @@
 
 . "$BASH_IT/themes/powerline/powerline.base.bash"
 
+CAN_SUDO=true
```

Comment:
If you set this here, how can the user override this value?

I suggest you change this to something like this, so it checks for an existing value, but uses `true` as the default:

```
THEME_CHECK_SUDO=${THEME_CHECK_SUDO:=true}
```